doc/source/shared/abbreviations.rst

```diff
@@ -30,6 +30,15 @@
 .. |N3LO| replace::
    :abbr:`N3LO (Next-to-Next-to-Next-to-Leading Order)`
 
+.. |LL| replace::
+   :abbr:`LL (Leading Power)`
```

Comment:
LL = leading log (meaning all logs have the correct power)
sometimes people use LP = leading power, e.g. to refer to the first power correction in the soft expansion (as people, e.g. in nikhef are doing :upside_down_face: )

Reply:
ops,  thanks for checking